test/bin/ui5.js

```diff
@@ -103,3 +103,47 @@ test.serial.cb("ui5 imports local installation when found (/w --verbose)", (t) =
 		t.end();
 	}, 0);
 });
+
+test.serial.cb("ui5 logs warning when using pre-release Node.js version", (t) => {
+	const consoleLogStub = sinon.stub(console, "log");
+
+	const importLocalStub = sinon.stub().returns(true);
+	mock("import-local", importLocalStub);
+
+	const sandbox = sinon.createSandbox();
```

Comment:
Is it required to create a sandbox here? I would expect that you can just use the default sandbox via `sinon.stub(..)` (didn't test)